test/cookbooks/firefox_package_test/recipes/default_controls.rb

```diff
@@ -1,43 +1,78 @@
-      expect(file('/usr/bin/firefox-latest')).to be_linked_to('/opt/firefox/latest_en-US/firefox')
-      expect(file('/opt/firefox/latest_en-US/firefox')).to be_executable
-    end
+    control 'latest' do
+      subject(:latest) { command('/opt/firefox/latest_en-US/firefox --version') }
+
+      it 'is installed and symlinked' do
+        expect(file('/usr/bin/firefox-latest')).to be_symlink
+        expect(file('/usr/bin/firefox-latest')).to be_linked_to('/opt/firefox/latest_en-US/firefox')
+        expect(file('/opt/firefox/latest_en-US/firefox')).to be_executable
+      end
 
-    it 'is functional when invoked' do
-      expect(latest.exit_status).to eq(0)
+      it 'is functional when invoked' do
+        expect(latest.exit_status).to eq(0)
+      end
     end
-  end
 
-  control '37.0' do
-    subject(:specified_version) { command('/opt/firefox/37.0_en-US/firefox --version') }
+    control '37.0' do
+      subject(:specified_version) { command('/opt/firefox/37.0_en-US/firefox --version') }
+
+      it 'is installed and symlinked' do
+        expect(file('/usr/bin/firefox-37.0')).to be_symlink
+        expect(file('/usr/bin/firefox-37.0')).to be_linked_to('/opt/firefox/37.0_en-US/firefox')
+        expect(file('/opt/firefox/37.0_en-US/firefox')).to be_executable
+      end
+
+      it 'is functional when invoked' do
+        expect(specified_version.exit_status).to eq(0)
+      end
 
-    it 'is installed and symlinked' do
-      expect(file('/usr/bin/firefox-37.0')).to be_symlink
-      expect(file('/usr/bin/firefox-37.0')).to be_linked_to('/opt/firefox/37.0_en-US/firefox')
-      expect(file('/opt/firefox/37.0_en-US/firefox')).to be_executable
+      it 'is the correct version of Firefox'do
+        expect(specified_version.stdout).to match(/Mozilla Firefox 37.0/)
+      end
     end
+  elsif os[:family] == 'windows'
+    control 'latest-esr' do
+      let(:bin_path) { 'C:\Program Files (x86)\Mozilla Firefox\latest-esr_en-US\firefox.exe' }
+      subject(:latest_esr) { command( "\"#{bin_path}\" --version") }
 
-    it 'is functional when invoked' do
-      expect(specified_version.exit_status).to eq(0)
+      it 'is installed' do
+        expect(file(bin_path)).to be_file
+      end
     end
 
-    it 'is the correct version of Firefox'do
-      expect(specified_version.stdout).to match(/Mozilla Firefox 37.0/)
+    control 'latest' do
+      let(:bin_path) {'C:\Program Files (x86)\Mozilla Firefox\latest_en-US\firefox.exe' }
+      subject(:latest_esr) { command( "\"#{bin_path}\" --version") }
+
+      it 'is installed' do
+        expect(file(bin_path)).to be_file
+      end
+    end
+
+    control '37.0' do
+      let(:bin_path) {'C:\Program Files (x86)\Mozilla Firefox\37.0_en-US\firefox.exe' }
+      subject(:specified_version) { command( "\"#{bin_path}\" --version") }
+
+      it 'is installed' do
+        expect(file(bin_path)).to be_file
+      end
+
+# This seems to be broken
+#      it 'is the correct version of Firefox'do
```

Comment:
You should be able to use `xit` instead of `it`. That way you can leave the test committed but it will be marked as pending.